src/settings.h

```diff
@@ -640,6 +642,12 @@ class settings : public QSettings, public singleton<settings> {
     };
     Q_ENUM(fake_keyboard_type_t)
 
+    enum class text_to_window_method_t {
+        CtrlV = 0,
+        Typing = 1
```

Comment:
```suggestion
        TextToWindowMethodCtrlV = 0,
        TextToWindowMethodTyping = 1
```

In QML, the enum scope is hidden, so this prefix makes everything more readable and less prone to errors.

Reply:
Great suggestion